index.html

```diff
@@ -857,11 +857,17 @@ <h4>Text</h4>
                 <a>Text</a> objects in one language can also contain some words in a different language.
                 This is represented in a <a>DAPT Document</a> by setting the <code>xml:lang</code> attribute on
                 inner <code>&lt;span&gt;</code> elements.</p>
+                <p class="note"><code>&lt;span&gt;</code> elements can be used to add specific styling (e.g. Japanese ruby annotations).</p>
+                <p class="note"><code>&lt;span&gt;</code> elements can be used to add specific timing
+                as illustrated in [[[#example-10]]] to indicate the timing of the audio rendering
+                of the relevant section of text. Per [[TTML2]], timing of the <code>&lt;span&gt;</code>
+                element is relative to the parent element's computed begin time.</p>
+                <p class="note">The text content of a <a>Text</a> object is the concatenated text content
+                of the <code>&lt;p&gt;</code> element and its descendent <code>&lt;span&gt;</code> elements.</p>
```

Comment:
Suggest moving these two up to the end of the first bullet in the list above - here they look like they might be something to do with `xml:lang`.